docs/diagrams/MarkTaskActivityDiagram.puml

```diff
@@ -1,17 +1,25 @@
 @startuml
 start
 :User enters mark [index] command;
-
-if () then ([invalid index])
-    :Display error message;
+:Input is parsed;
+if () then ([index is not an interger])
```

Comment:
Perhaps combine all the branches into a single diamond with multiple branches?

Reply:
The if-else branches are staggered to reflect that the checks are done one after another.